packages/config-loader/src/sources/ConfigSources.ts

```diff
@@ -71,6 +72,7 @@ export interface ClosableConfig extends Config {
  * @public
  */
 export interface BaseConfigSourcesOptions {
+  watch?: LoadConfigOptionsWatch;
```

Comment:
```suggestion
  watch?: boolean;
```

Realized this is the wrong type